### ISSUE TITLE
[Issue]: Swiss Fails to Initialise DVD Drive

### ISSUE THREAD

**Issue by Knightsurfer** [emukidid/swiss-gc#855]
### Describe the bug

This was working fine before I updated, no idea what version I had before.

### Steps to reproduce

I do the smash exploit, normally I just have to wait for the broadband adapter to initialise but this time I apparently have to wait on the dvd drive too, only for it to fail, applications such as cleanrip recognise the drive just fine.

### Expected behavior

I expected to be able to read the contents of the dvd.

**Comment by Knightsurfer** [comment]
upon further investigation I still have an earlier version of swiss on one of my storage devices, that is version 1443 , another weird thing I have found is the bug only shows up when using the exploit, going from homebrew to homebrew makes it initialise just fine.

**Comment by Extrems** [comment]
Duplicate of issue #843.
